sdk/ml/azure-ai-ml/azure/ai/ml/_schema/compute/setup_scripts.py

```diff
@@ -2,6 +2,7 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # ---------------------------------------------------------
 
+# pylint: disable=unused-argument,no-self-use
```

Comment:
What this a pylint issue that was always present, but never caught until you editted nearby files?

Reply:
Yes -- not sure why it was never caught in the setup_scripts file, as it has the same structure as the new custom_application file. But the rule is present in other similar files, like _schema//compute_instance.py.